src/assignments/launch-checklist.rst

```diff
@@ -0,0 +1,78 @@
+Assignment #5: Launch Checklist Form
+====================================
+
+Using our knowledge of forms, the DOM, and HTTP, our favorite space shuttle program has asked us to create a quick launch checklist.
+We have four fields that need to be filled out with vital information: the pilot's name, the co-pilot's name, the fuel levels, and the weight of the cargo.
+
+Our pilot, Chris, and their co-pilot, Blake, have been hard at work securing the cargo and filling the shuttle tank. All we need to do is use validation to insure that we have all of the info for the space shuttle program and make sure no one prematurely launches the shuttle.
+
+Requirements
+------------
+
+Create a Launch Checklist Form for astronauts to fill out in preparation for launch.
+The form should do the following:
+
+1. Validate the user responses with ``preventDefault()`` to insure the following:
+
+   1. The user has entered something for every field.
+   2. The user has entered text for names and numbers for fuel and cargo levels.
+
+2. With validation, update a listing of what is currently ready for shuttle launch and not.
+3. Use CSS to indicate what is good and bad about the shuttle and whether it is ready for launch. 
```

Comment:
This seems to contradict the note about not changing the CSS file.

Reply:
They have the DOM! I will add some clarity to this point that they should change CSS through the DOM.

Reply:
Thanks!